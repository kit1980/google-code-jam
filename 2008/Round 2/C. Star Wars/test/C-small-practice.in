10
3
1 0 0 1
2 1 1 4
3 2 3 2
10
749709 772910 278417 6
208544 311102 354951 9
698123 952054 988819 1
252486 827293 503163 2
730571 757552 677587 8
492834 423148 364526 8
746326 533367 349434 9
896852 717974 338577 10
36714 843765 227839 11
862207 802188 940476 6
10
10836 7165 2791 3
6035 11005 8515 1
18109 4297 9143 3
18502 20752 4148 5
14907 7718 1803 3
31183 2024 2515 5
5946 18896 3861 3
3976 4118 12647 3
8955 4554 594 6
16981 10269 1222 3
7
39489 18192 101846 3
67760 6425 97354 1
16169 12534 93019 6
26398 26123 102866 4
12032 8384 102916 6
53473 40672 89250 3
58811 24255 91307 3
4
0 0 0 1
1 2 0 1
3 4 0 1
2 1 0 1
10
4123 4407 1235 5
12436 17594 2680 1
23229 8101 1123 4
11478 15131 3761 5
15120 2543 11612 4
23642 12123 4690 3
6330 18025 3076 5
860 6508 6308 4
20876 17895 4970 5
1315 6894 3728 4
9
20 20 0 1
20 10 0 2
10 20 0 3
20 0 0 2
10 0 0 3
10 10 0 1
0 0 0 3
0 20 0 1
0 10 0 2
10
89 59 65 4
89 11 66 3
86 17 22 2
32 33 55 1
45 97 81 4
38 79 18 3
17 91 16 3
30 34 7 1
23 40 44 3
0 73 12 4
6
15543 17150 2162 1324
27809 9767 3575 1324
14241 17379 7798 1324
3873 14256 11238 1324
6222 1004 5769 1324
26100 16937 3921 1324
8
10 0 10 1
0 10 10 3
0 0 10 2
0 10 0 1
0 0 0 2
10 10 10 1
10 10 0 1
10 0 0 3
