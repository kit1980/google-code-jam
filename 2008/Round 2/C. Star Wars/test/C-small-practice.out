Case #1: 2.33333333333333
Case #2: 352018.0
Case #3: 6688.16666666667
Case #4: 14159.5
Case #5: 3.5
Case #6: 5333.4
Case #7: 13.3333333333333
Case #8: 35.25
Case #9: 14.2216767371601
Case #10: 15.0
